src/ethoscope/core/roi.py

```diff
@@ -158,4 +164,11 @@ def apply(self,img):
         if out.shape[0:2] != self._mask.shape:
             raise EthoscopeException("Error whilst slicing region of interest. Possibly, the region out of the image: %s" % str(self.get_feature_dict()), img )
 
-        return out, self._mask
\ No newline at end of file
+        return out, self._mask
+
+    def find_region(self, x, y):
+        try:
+            gray_value = self._regions[x, y]
+        except:
```

Comment:
always print exception as a warning:
```
 try:
    gray_value = self._regions[x, y]
except Exception as e:
    logging.warning(e)
    ...
```
Also you can specify the type of exceptions `Expect IndexError as e`.